src/pybind/mgr/cephadm/module.py

```diff
@@ -1252,6 +1277,24 @@ def _refresh_host_devices(self, host):
         self.cache.save_host(host)
         return None
 
+    def _deploy_etc_ceph_ceph_conf(self, host: str) -> Optional[str]:
+        try:
+            with self._remote_connection(host) as tpl:
+                conn, connr = tpl
+                out, err, code = remoto.process.check(
+                    conn,
+                    ['mkdir', '-p', '/etc/ceph'])
+                if code:
+                    return f'failed to create /etc/ceph on {host}: {err}'
+                out, err, code = remoto.process.check(
+                    conn,
+                    ['dd', 'of=/etc/ceph/ceph.conf'])
```

Comment:
maybe I don't see it, but where are we getting the file content? stdin?

Reply:
that's why I said in the standup, this PR as of right now, doesn't do anything. Note the PR is still [WIP]